packages/ramps-controller/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/ramps-controller",
-  "version": "1.0.0",
+  "version": "2.0.0",
```

Comment:
Our release automation in this repo is setup to handle version bumps in release branches only. We can't change versions outside of release branches, it would end up breaking when we tried to release this.

Reply:
Good to know, thanks. I've reverted this update.